yang/ietf-schedule.yang

```diff
@@ -229,26 +229,33 @@ module ietf-schedule {
   identity enabled {
     base schedule-state;
     description
-      "Indicates a recurrence with an enabled state.";
+      "Indicates a schedule with an enabled state.";
```

Comment:
This change is consistent with the use in the draft. However, there might be cases where we may need to track the state of a recurrence. For example, a global schedule can be enabled but a given occurrence skipped/disabled/conflict/etc. The use of distinct identities would allow to customize allowed values.

Reply:
Yes, agree. but this might be indicated by the value of "schedule-type" sibling parameter: if the schedule-type is recurrence, then  the state parameter indicates the state of recurrence instances, otherwise, it indicates the state of a particular single period. Make sense?

Reply:
That would work unless we find a case where the state value of recurrence do not apply for schedules (and vice versa). On the other hand, having distinct identities (although same set values are defined in the doc) allows for better flexibility and future uses where state may deviate between the two. 

If we maintain one single id, then we need to update the description to call this out. Also, update the narrative text to describe the intended behavior.

The approach you will pick, will be mine :-)

Reply:
So the question is, for a recurrence rule with a set of occurrences, whether the state parameter indicates the existing occurrence status or the entire schedule status—they are not always the same.

For a whole enabled schedule with a couple of conflicted/disabled/etc. occurrences cases, I am thinking there are some other parameters like failure-counter to indicate (but only a numeric value, not which of them).

Just make sure we are on the same page:
When you say "having distinct identities", are you referring to have another parameter like existing-occurrence-state definition? What if you're in the gap between two occurrences?

When you say "update the description to call this out", are you referring to describe the state of global recurrence only?

Reply:
> When you say "having distinct identities", are you referring to have another parameter like existing-occurrence-state definition? What if you're in the gap between two occurrences?

Yes for the first question. 
For the second question, the same concern applies independent of whether the same or distinct state values are used.

> When you say "update the description to call this out", are you referring to describe the state of global recurrence only?

I meant, update the description to mention "schedule (or recurrence)"

Reply:
> I meant, update the description to mention "schedule (or recurrence)"
Just checked the existing description and narrative text, all are speaking about schedule already, which I think should be fine now.